### PR TITLE
Add simple producer consumer load test for kafka

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
## Examples
N/A
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/2143

### CODE REVIEW

File: load-tests/simple_producer_consumer/scripts/load_test/main.bal [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+import ballerina/http;
+import ballerina/lang.runtime;
+import ballerina/io;
+import ballerina/time;
+
+public function main(string label, int users, string output_csv_path) returns error? {
```
Comment (shafreenAnfar) [inline load-tests/simple_producer_consumer/scripts/load_test/main.bal:23]: What does this `users` argument do ?

Comment (dilanSachi) [reply load-tests/simple_producer_consumer/scripts/load_test/main.bal:23]: The result csv contains the user count. For now, "1" is passed for this

Comment (shafreenAnfar) [reply load-tests/simple_producer_consumer/scripts/load_test/main.bal:23]: So if we pass 2, does it create 2 producers ?

Comment (dilanSachi) [reply load-tests/simple_producer_consumer/scripts/load_test/main.bal:23]: no
this is just to create the csv file. it does not affect the kafka clients

Comment (shafreenAnfar) [reply load-tests/simple_producer_consumer/scripts/load_test/main.bal:23]: In that case, lets remove it from the arguments.

**Review by shafreenAnfar — APPROVED** [review]

**Pull request merged**